hybrasyl/Objects/Creature.cs

```diff
@@ -714,7 +714,7 @@ public virtual bool Walk(Direction direction)
             Direction = direction;
 
             // Objects in the common viewport receive a "walk" (0x0C) packet
-            // Objects in the arriving viewport receive a "show to" (0x33) packet
+            // Objects in the arriving viewport receive a "show to" (_) packet
```

Comment:
???????

Reply:
yeah idk. Fixed

Reply:
I wasn't sure if you figured out something new.